airflow/api_connexion/openapi/v1.yaml

```
@@ -2238,6 +2238,9 @@ components:
         conn_type:
           type: string
           description: The connection type.
+        description:
+          type: string
+          description: The description of the connection.
```

```suggestion
          description: |
              The description of the connection.
              
              *New in version 2.4.0*
          nullable: true
```
We can add nullable for both descriptions

---

The *New in version 2.4.0* is missing

---

Ah but it isn't new in 2.4. It was there before, just not documented :)